src/parser/ad_parser.js

```diff
@@ -182,12 +185,17 @@ function parseAdElement(adTypeElement, emit) {
       case 'BlockedAdCategories':
         ad.blockedAdCategories.push({
           authority: node.getAttribute('authority') || null,
-          value: parserUtils.parseNodeText(node)
+          value: parserUtils.parseNodeText(node),
         });
         break;
     }
   }
 
+  if (adVerificationsFromExtensions.length && ad.system.version < 4.1) {
```

Comment:
So we stick to the spec and only take adVerification from extension if VAST version is < 4.1
2 question 
- is `system.version` is a number?
- Should we have this restriction with the version (it's an open question)

Reply:
Hey Rumesh, thank you for your comment, you were right, system.version is a string and as an absolute, we shouldn't leave the browser to deal with the conversion. 

I'll remove this condition anyway because as we are retrocompatible, I don't think we need to be strict with this condition.